### PR TITLE
zscroll: init at 1.0

### PR DESCRIPTION
**Pull request by noctuid**
I'm quite new to nix, and this is my first attempt at a package. I tested it, and it built and worked correctly.

### CODE REVIEW

**Comment by noctuid** [comment]
Oops, I see that I accidentally left ${name} in there, making the url invalid. I will fix this.

File: pkgs/applications/misc/zscroll/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, python3, python3Packages, fetchurl }:
+
+let version = "1.0"; in
+
+python3Packages.buildPythonPackage {
+  name = "zscroll-${version}";
+  # don't prefix with python version
+  namePrefix = "";
+
+  src = fetchurl {
```
Comment (pSub) [inline pkgs/applications/misc/zscroll/default.nix]: Please use `fetchFromGitHub`.

File: pkgs/applications/misc/zscroll/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    sha256 = "0lfca7prg5vih9064r3xwss8f4x7zpdj1m44b4lw2dys6rxl4lxl";
+  };
+
+  doCheck = false;
+
+  propogatedBuildInputs = [ python3 ];
+
+  meta = with stdenv.lib; {
+    description = "A text scroller for use with panels and shells";
+    homepage = "https://github.com/noctuid/zscroll";
```
Comment (pSub) [inline pkgs/applications/misc/zscroll/default.nix]: Nitpick: You don't need quotes around URLs.

**Comment by jagajaga** [comment]
Please don't forget to squash commits and properly name as is said in `CONTRIBUTING.md`

**Comment by noctuid** [comment]
@jagajaga Okay, I'll do that. Thank you.

**Pull request merged**